app/src/main/java/com/katiearose/sobriety/utils/Utils.kt

```diff
@@ -9,13 +9,18 @@ import androidx.preference.PreferenceManager
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import com.katiearose.sobriety.R
 import java.time.Instant
+import java.time.LocalDateTime
+import java.time.Period
+import java.time.Duration
+import java.util.TimeZone
 
 private const val MINUTE = 60
 private const val HOUR = MINUTE * 60
 private const val DAY = HOUR * 24
 private const val WEEK = DAY * 7
-private const val MONTH = DAY * 31
-private const val YEAR = MONTH * 12
+private const val YEAR = DAY * 365
```

Comment:
Nitpicking: This should be `365.2425` to account leap years :smirk: 

(every 4 years, you have 1 leap year, with the exception for every 100 years where no leap year exists, but for every 400 years this doesn't hold, so we have `(100 / 4 - 1) * 4 + 1` = 97 leap years every 400 years => `97/400` = `0.2425`)